plugin/evm/config.go

```diff
@@ -24,8 +25,11 @@ const (
 	defaultMaxBlocksPerRequest                  = 0 // Default to no maximum on the number of blocks per getLogs request
 	defaultContinuousProfilerFrequency          = 15 * time.Minute
 	defaultContinuousProfilerMaxFiles           = 5
-	defaultTxRegossipFrequency                  = 1 * time.Minute
-	defaultTxRegossipMaxSize                    = 15
+	defaultTxRegossipFrequency                  = 1 * time.Second
```

Comment:
Should this stay at 1 minute and only use 1 second as the default for priority re-gossip?

Reply:
Good catch